package.json

```diff
@@ -71,11 +72,11 @@
     "@paypal/react-paypal-js": "^8.7.0",
     "@radix-ui/react-form": "^0.1.0",
     "@radix-ui/react-tooltip": "^1.1.2",
-    "@sentry/browser": "^8.42.0",
-    "@sentry/nestjs": "^8.42.0",
-    "@sentry/nextjs": "^8.42.0",
-    "@sentry/node": "^8.42.0",
-    "@sentry/opentelemetry": "^8.42.0",
+    "@sentry/browser": "8.42.0",
```

Comment:
For these package groups, I'm a fan of pinning versions! I've seen things break because these get out sync. 👍